assets/css/app.scss

```diff
@@ -86,6 +86,19 @@ $z-properties-panel-context: (
 @import "vehicle_map";
 @import "view_header";
 
+// Bootstrap variable customization
+$component-active-bg: $color-eggplant-700;
```

Comment:
question (non-blocking): I may be missing something - wouldn't this get overridden by the bootstrap imports below?

Reply:
Oh very cool! That section makes it seem like the `/functions` import should come first, but I can't tell if that actually is required for the color override to work.

> Variable overrides must come after our functions are imported, but before the rest of the imports.